Moonscraper Chart Editor/Assets/Scripts/Game/Controllers/Song Objects/DrumRollController.cs

```diff
@@ -17,6 +17,9 @@ public class DrumRollController : SongObjectController
     [SerializeField]
     GameObject[] m_laneVisuals;
 
+    [SerializeField]
+    public SustainResources resources;
```

Comment:
No need to make public
```suggestion
    [SerializeField]
    SustainResources resources;
```